packages/edit-site/src/components/preferences-modal/index.js

```diff
@@ -86,6 +86,13 @@ export default function EditSitePreferencesModal() {
 						) }
 						label={ __( 'Display block breadcrumbs' ) }
 					/>
+					<EnableFeature
+						featureName="allowRightClickOverrides"
```

Comment:
To me, `enableListViewRightClick` or `allowListViewContextualMenus` or the equivalent specific descriptor might describe the feature. `allowRightClickOverrides` might be too generic?

Reply:
Good question! My thinking was to keep it generic so that if we wish to expand other areas to also have right click overrides, we don't wind up with a proliferation of `allowRightClick` settings in the preferences 🤔